### ISSUE TITLE
[BUG]: colossalai.get_default_parser()

### ISSUE THREAD

**Issue by jinlovespho** [hpcaitech/ColossalAI#5475]
### 🐛 Describe the bug

okay I am new to parallel distributed training. I was following along the basic tutorials provided in https://colossalai.org/docs/basics/launch_colossalai. 
![SmartSelectImage_2024-03-20-14-24-12](https://github.com/hpcaitech/ColossalAI/assets/95962130/84846c63-a8c5-4acc-ae60-409db49c3afd)

I am currently trying to check => parser = colossalai.get_default_parser()
But apparently I'm getting AttributeError: module 'colossalai' has no attribute 'get_default_parser'.
I searched this issue but found nothing, I guess everyone else is not going through this problem.. 
Can somebody help? Thanks in advance !

**Comment by thebigbone** [comment]
Same error. Seems like it's not imported correctly because there is a function called ```get_default_parser``` in the project directory.

**Comment by char-1ee** [comment]
@thebigbone Yes, but the `colossalai.legacy` module is now deprecated. You can refer to the new initialization methods in https://github.com/hpcaitech/ColossalAI/blob/main/colossalai/initialize.py, which you might need to parse arguments on your own and pass in `colossal.initialize.launch` to start a distributed training environment.
